resources.md

```diff
@@ -21,4 +21,4 @@ The Open Forcefield Group makes a number of resources available to the community
 
 ## Forcefields
 
-- [smirnoff99Frosst](https://github.com/open-forcefield-group/smirnoff99Frosst): The first general-purpose implementation of a SMIRKS Native Open Force Field (SMIRNOFF) as implemented by SMARTY and its ForceField class (in smarty.forcefield) for parameterizing small molecules for OpenMM.
+- [smirnoff99Frosst](https://github.com/open-forcefield-group/smirnoff99Frosst): The first general-purpose implementation of a SMIRKS Native Open Force Field (SMIRNOFF) as implemented by the SMIRNOFF ForceField class (in openforcefield.typing.engines.forcefield) for parameterizing small molecules for OpenMM and (via ParmEd and InterMol) a wide variety of other simulation packages.
```

Comment:
Protect `openforcefield.typing.engines.forcefield` with backticks so it renders correctly?

Link to [ParmEd](https://github.com/parmed/parmed) and [InterMol](https://github.com/shirtsgroup/InterMol)?